daemon/daemon.go

```diff
@@ -96,7 +97,7 @@ func (d *Daemon) ListServices(namespace string) ([]flux.ServiceStatus, error) {
 	return res, nil
 }
 
-// List the images available for set of services
+// ListImages - List the images available for set of services
```

Comment:
Are these a result of linting or something?

Reply:
(this is ignorable, I just find those changes a bit distracting so I wondered what produced them)

Reply:
It's recommended that public methods/functions are commented and the comment starts with the method/function name. I am using Visual Studio and it is >helpfully<  highlighting all that does not have the recommended documentation. So, sometimes, I update the comment.

Reply:
FWIW I think this should be `ListImages lists the images ...`

Reply:
I think both options are acceptable. A matter of personal preference I would say.
FYI, the implementation is being refactored.